src/ostorlab/utils/reporter.py

```diff
@@ -0,0 +1,53 @@
+"""Reporter daemon logic that report scanner state periodically."""
```

Comment:
Reporter logic to read the scanner state periodically and send it to the backend.